src/Checkbox/Checkbox.example.js

```diff
@@ -146,4 +147,27 @@ storiesOf('Checkbox', module)
         propTables: [Checkbox],
       },
     },
+  )
+  .add(
+    'With disabled props',
+    () => (
+      <ThemeProvider theme={theme}>
```

Comment:
remove `ThemeProvider` (already setup ThemeProvider in `./storybook/decorator.js`)